File: ShowHideUI/app/src/main/java/com/example/showhideui/MainActivity.java
package com.example.showhideui;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    TextView text ;
    public void show(View view){
        text = (TextView) findViewById( R.id.textView);
        text.setVisibility(View.VISIBLE);
    }
    public void hide(View view){
        text = (TextView) findViewById( R.id.textView);
        text.setVisibility(View.INVISIBLE);
    }
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        text = (TextView) findViewById( R.id.textView);

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);



    }
}

File: KotlinDemo/app/src/main/java/com/example/kotlindemo/MainActivity.kt
package com.example.kotlindemo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    var count = 0
    fun reset(view : View){
        count = 0
        textView.text = count.toString()
    }
    fun addone(view: View){
        count ++
        textView.text = count.toString()
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var textView = findViewById<TextView>(R.id.textView)
        textView.text = "0"
    }
}
